src/migrations/20201120182130-create-role.js

```diff
@@ -3,14 +3,16 @@ module.exports = {
     await queryInterface.createTable('Roles', {
       id: {
         allowNull: false,
-        autoIncrement: false,
+        autoIncrement: true,
```

Comment:
Do we need to wipe dbs that have already had the previous create-role migration run to pick up this change?

Reply:
Yes. We need to undo all migrations and run them again.

Reply:
is `terraform destroy && terraform apply` the best way to accomplish that at this point? cc/ @SarahJaine

Reply:
What environments are we talking about doing this in? We just need to do this the **_first_** time this migration is run, correct?
--
To answer your question, to blow away and restore dev rds instance you'd need to do
1. from `terraform/dev` run `terraform -destroy -target=cloudfoundry_service_instance.database` (target the db, there's no need to destroy the s3 bucket too)
2. respond to confirmation prompt
3. `terraform apply`
4. respond to confirmation prompt
5. Then I think you might need to rerun circleci `build_release_deploy` workflow on the `main` branch in circle ci to deploy the app and reinstate the "binding" between the rds instance and the application. You can first check if it's still there by looking at the services in cloud.gov.

But I think you're right to blow away the whole rds instance. I tried to just run `db:drop` and `db:create` directly against a our sandbox db last week and I couldn't get `db:create` to run. Then again, I tried for like 5 minutes before moving on and just destroying/reapplying the whole rds instance.

Reply:
It'll need to happen in any environments that this migration had run against already before being edited, so that's `staging` and `dev` at least. It won't need to happen on `prod` because prod will only see this updated version the first time it's run. Once we have data that needs to be preserved the process will be to create new migrations for any changes, rather than editing old migraitons.

And yeah - a new deploy to redo the binding (and also to re-run the migrations) would be needed.

Reply:
Oh okay, we're changing a migration that's already run- sorry, I'm all caught up now. In sandbox and sandbox only, I added a `migrate:undo` step before migrations run (you'll see it in a forthcoming PR). I was pretty much banking on dev/staging/prod not having edited migrations, so I didn't add this to any other environment. Since we don't have any data in these systems yet blowing them away is probably the most straightforward approach.